components/Api/Stability/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { PropsWithChildren, FC } from 'react';
+
+type StabilityProps = PropsWithChildren<{ stability: number }>;
+
+const getStabilityClass = (stability: number) => {
+  switch (stability) {
+    case 0:
+      return styles.stabilityLevel0;
+    case 1:
+      return styles.stabilityLevel1;
+    case 2:
+      return styles.stabilityLevel2;
+    case 3:
+      return styles.stabilityLevel3;
+    default:
+      throw new Error(`Unknown stability level: ${stability}`);
+  }
+};
```

Comment:
Can we optimise this function?


I think, its better to reduce the code as below:

```suggestion
 const getStabilityClass = (stability: number) => {
    const style = styles[`stabilityLevel${stability}`]
    if(!style)
        throw new Error(`Unknown stability level: ${stability}`);
 };
```